Common/Extensions/Bundle+LoopKit.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Bundle.swift
+//  Loop
+//
+//  Created by Michael Pangburn on 12/27/18.
+//  Copyright © 2018 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import LoopKit
+import LoopKitUI
+
+
+extension Bundle {
+    static let linkedPumpManagers = linkedFrameworkClasses(fromInfoDictionaryKey: "PumpManager").compactMap { $0 as? PumpManager.Type }
+    static let linkedCGMManagers = linkedFrameworkClasses(fromInfoDictionaryKey: "CGMManager").compactMap { $0 as? CGMManager.Type }
+
+    private static func linkedFrameworkClasses(fromInfoDictionaryKey plistKey: String) -> [AnyClass] {
```

Comment:
Clever. I like this idea.

Reply:
I might make this function public and move `linkedPumpManagers` and `linkedCGMManagers` into locations that are specific to PumpManagers and CGMManagers.

When I see an extension I ask myself "is this generally useful to other users of the thing being extended?" It seems like `linkedFrameworkClasses` is a general Bundle-related function that could be used in all sorts of Bundle situations. However, for most users of the general Bundle class, the `linkedPumpManagers` and `linkedCGMManagers` make no sense. However, `linkedPumpManagers` and `linkedCGMManagers` would make total sense in `DeviceDataManager` or similar location.

Or am I missing something? Thoughts?